src/Workspaces/MSBuildTest/WorkspaceTestBase.cs

```diff
@@ -107,6 +107,10 @@ protected static FileSet GetSimpleCSharpSolutionFiles()
         protected static FileSet GetSimpleVisualBasicSolutionFiles()
         {
             return new FileSet(
+                (@"NuGet.Config", Resources.NuGet_Config),
+                (@"Directory.Build.props", Resources.Directory_Build_props),
+                (@"Directory.Build.targets", Resources.Directory_Build_targets),
+                (@"TestSolution.sln", Resources.SolutionFiles.VisualBasic),
```

Comment:
It took me a good 3 minutes to figure out that this is actually `VisualBasic.sln` that was added in this PR. Guessing other devs in the future may have a similar struggle. Did you consider having the names match to make the connection easier?

Reply:
@jaredpar: this is how it works elsewhere too:

https://github.com/dotnet/roslyn/blob/bf23384bd13c3e95e6cc4a584bebc16960e55709/src/Workspaces/MSBuildTest/WorkspaceTestBase.cs#L101
https://github.com/dotnet/roslyn/blob/bf23384bd13c3e95e6cc4a584bebc16960e55709/src/Workspaces/MSBuildTest/WorkspaceTestBase.cs#L221

I agree this is confusing, but it'd require changing all the other tests to make this be consistent.

Reply:
Sigh ... :shipit: